extensions/amp-accordion/0.1/amp-accordion.js

```diff
@@ -32,7 +32,7 @@ class AmpAccordion extends AMP.BaseElement {
     this.sections_ = this.getRealChildren();
 
     /** @const @private {!Window} */
-    this.win_ = this.getWin();
+    this.win_ = this.win;
```

Comment:
Grrr. `this.win_` should probably be removed completely. Ok to pass on this for this PR though.